x/commerciodocs/querier.go

```diff
@@ -75,13 +76,13 @@ func queryGetAuthorized(ctx sdk.Context, path []string, keeper Keeper) (res []by
 
 	bz, err2 := codec.MarshalJSONIndent(keeper.cdc, connectionsResult)
 	if err2 != nil {
-		panic("Could not marshal result to JSON")
+		return nil, sdk.ErrUnknownRequest(fmt.Sprintf("Could not marshal result to JSON"))
```

Comment:
If we do not have any error variables I think we can remove the usage of `fmt.Sprintf` and leave the play string.